src/main/java/com/github/manolo8/darkbot/gui/login/LoginForm.java

```diff
@@ -100,7 +101,13 @@ protected LoginData doInBackground() {
                 LoginUtils.findPreloader(loginData);
                 return loginData;
             } catch (Exception e) {
-                publish(new Message(true, "Failed to login!", IssueHandler.createDescription(e)));
+                if (e instanceof SSLHandshakeException) //?
+                    publish(new Message(true, "Too old Java!", IssueHandler.createDescription(e)));
+                else if (e instanceof LoginUtils.LoginException)
+                    publish(new Message(true, ((LoginUtils.LoginException) e).titleMessage, IssueHandler.createDescription(e)));
+                else
+                    publish(new Message(true, "Failed to login!", IssueHandler.createDescription(e)));
+
```

Comment:
Use multiple-catch expressions, one for ssl, one for login exception, and then a final one for just Exception

Also for titleMessage use a getter (ie: getTitle() )